libraries/chain/protocol/asset.cpp

```diff
@@ -97,15 +97,19 @@ namespace graphene { namespace chain {
          const asset& a = *this;
          if( a.asset_id == b.base.asset_id )
          {
+            // TODO replace exception type with price_multiplication_undefined?
```

Comment:
Please remove these unrelated comments. If you think this might be a good idea, open a github issue.

Reply:
Just realized that the definitions of those exceptions are commented out. Will remove the unrelated comments.

Reply:
Removed.